examples/covid_report/main.bal

```diff
@@ -0,0 +1,42 @@
+import nuvindu/pipe;
+import ballerina/io;
+import ballerina/log;
+import ballerina/lang.runtime;
+//io:fprintln, log:printError
+public function main() returns error? {
+    pipe:Pipe pipe = new(5);
+    Report[] reports = check getReportData();
+    worker A {
+        foreach Report report in reports  {
+            io:println("starts");
+            error? produce = pipe.produce(report, timeout = 5);
+            io:println("produces");
+            if produce is error {
+                io:println(produce);
+            }
+            runtime:sleep(1);
+        }
+        error? gracefulClose = pipe.gracefulClose();
+        if gracefulClose is error {
+            log:printError(gracefulClose.message(), gracefulClose); 
+        }
+    }
+
+    @strand {
+        thread: "any"
+    }
+    worker B {
+        stream<Report, error?> covidReports = pipe.consumeStream(timeout = 5.12323);
+        CovidRecord|error? covidRecord = covidReports.next();
+        int i = 0;
+        while covidRecord is CovidRecord {
+            Report covidReport = covidRecord.value;
+            io:println("Date: ",covidReport.date);
```

Comment:
There should be a space after the previous parameter. Shall we try to format the code using the VSCode formatter?

```suggestion
            io:println("Date: ", covidReport.date);
```